packages/vaadin-login/test/login-overlay.test.js

```diff
@@ -97,6 +97,23 @@ describe('opened overlay', () => {
     pressEnter(vaadinLoginUsername);
     expect(submitSpy.called).to.be.false;
   });
+  it('should focus the username field', () => {
+    const usernameElement = overlay.$.vaadinLoginForm.$.vaadinLoginUsername;
+    expect(document.activeElement).to.equal(usernameElement.inputElement);
+  });
+});
+
+describe('no autofocus', () => {
+  let activeElement;
+
+  beforeEach(() => {
+    activeElement = document.activeElement;
+    fixtureSync('<vaadin-login-overlay></vaadin-login-overlay>');
```

Comment:
This suite does not have `no-autofocus` while the test suite title says the opposite.

Reply:
Fixed test